internal/oci/runtime_vm.go

```diff
@@ -812,31 +803,21 @@ func metricsToCtrStats(ctx context.Context, c *Container, m *cgroups.Metrics) *C
 				m.Memory.TotalInactiveFile, memUsage,
 			)
 		}
-
-		for _, entry := range m.Blkio.IoServiceBytesRecursive {
-			switch strings.ToLower(entry.Op) {
-			case "read":
-				blockInput += entry.Value
-			case "write":
-				blockOutput += entry.Value
-			}
-		}
+		rssBytes = m.Memory.RSS
+		pageFaults = m.Memory.PgFault
+		majorPageFaults = m.Memory.PgMajFault
 	}
 
 	return &ContainerStats{
-		BlockInput:      blockInput,
-		BlockOutput:     blockOutput,
 		Container:       c.ID(),
-		CPU:             cpu,
 		CPUNano:         cpuNano,
-		MemLimit:        memLimit,
 		MemUsage:        memUsage,
-		MemPerc:         memPerc,
-		NetInput:        netInput,
-		NetOutput:       netOutput,
```

Comment:
@haircommander, I feel like our future selves would appreciate a comment, probably in the commit message, explaining why CPU*, Net*, and Block* stats coming from the containers are not relevant anymore.

Reply:
the thing is they have never been relevant. I think they were added because they did come from the cgroup package we use but it does feel odd to leave a comment about code that was dead and wasteful